docs/pages/_app.js

```diff
@@ -121,13 +121,9 @@ function forcePageReload(registration) {
 }
 
 async function registerServiceWorker() {
-  if (
-    'serviceWorker' in navigator &&
-    process.env.NODE_ENV === 'production' &&
-    window.location.host.indexOf('mui.com') !== -1
```

Comment:
I removed this line just for testing purposes in the deploy preview.
Looks good:
<img width="607" src="https://github.com/mui/mui-x/assets/13808724/fdbd8555-c952-4525-9d18-563d321ced28">

I'll revert this change